src/reducers/usersReducer.js

```diff
@@ -0,0 +1,48 @@
+import initialState from '../store/initialState';
+import {
+  REGISTER_REQUEST,
+  REGISTER_FAILURE,
+  REGISTER_SUCCESS,
+  VERIFY_USER
+} from '../actions/actionTypes';
+
+const { user } = initialState;
+const userReducer = (state = user, action) => {
+  const { type } = action;
+  switch (type) {
+    case REGISTER_REQUEST: {
+      return {
+        ...state,
+        isLoading: false,
```

Comment:
I don't think there's any need to have this in the userReducer since we have a globalReducer

Reply:
Feedback duly observed and implemented